docs/BR.md

```diff
@@ -2312,6 +2313,10 @@ Prior to including a Reserved Certificate Policy Identifier, the CA MUST ensure
    If a CRL entry is for a Certificate subject to these Requirements, the `CRLReason` MUST NOT be certificateHold (6).
 
    If a `reasonCode` CRL entry extension is present, the `CRLReason` MUST indicate the most appropriate reason for revocation of the certificate, as defined by the CA within its CP/CPS.
+   
+2. `issuingDistributionPoint` (OID 2.5.29.28)
+
+   Effective 2023-01-15, if a CRL does not contain entries for all revoked unexpired certificates issued by the CRL issuer, then it MUST contain a critical Issuing Distribution Point extension and MUST populate the `distributionPoint` field of that extension.
```

Comment:
It would be helpful to include the specification for the `distributionPoint` field value directly or through reference to [RFC 5280 §4.2.1.13](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.13).

Reply:
I am following the example of 7.2.2.1 `reasonCode` above, which simply makes statements about when the Reason Code field must appear and what values it can hold.